frontend/packages/console-shared/src/components/pod/__tests__/PodRingSet.spec.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 import { PodKind } from '@console/internal/module/k8s';
 import { DeploymentConfigModel } from '@console/internal/models';
-import { samplePods } from '@console/dev-console/src/components/topology/__tests__/topology-test-data';
+import { samplePods } from '@console/dev-console/src/components/topology/__mocks__/topology-test.mock';
```

Comment:
@vikram-raj `__mocks__` folder is for jest mocks https://jestjs.io/docs/en/manual-mocks
Keep test data in the `__tests__` dir or re-usable in the `src/test/data` dir.

We should also avoid having `console-shared` using `dev-console` imports. Not for this PR, but please create a new issue to track breaking this dependency.

Reply:
@christianvogt updated it and created a task to refactor it https://issues.redhat.com/browse/ODC-2636